src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1885,14 +1885,12 @@ internal enum ErrorCode
         WRN_UseDefViolationOut = 8886,
         WRN_UseDefViolation = 8887,
 
-        #endregion diagnostics introduced for C# 9.0
-
-        // PROTOTYPE(func-ptr): Pack before merge
+        ERR_CannotSpecifyManagedWithUnmanagedSpecifiers = 8888,
+        ERR_RuntimeDoesNotSupportUnmanagedDefaultCallConv = 8889,
+        ERR_TypeNotFound = 8890,
+        ERR_TypeMustBePublic = 8891,
 
-        ERR_CannotSpecifyManagedWithUnmanagedSpecifiers = 9500,
-        ERR_RuntimeDoesNotSupportUnmanagedDefaultCallConv = 9501,
-        ERR_TypeNotFound = 9502,
-        ERR_TypeMustBePublic = 9503,
+        #endregion diagnostics introduced for C# 9.0
```

Comment:
FYI I'm preparing a PR that attaches C# 9 features to LangVer 9 and updates tests to use LangVer 9 instead of Preview. 
Roughly when are we planning to merge the feature branch into master?
I'd recommend that you update function pointers similarly before merging to master, to reduce potential timing issues.